_docs/contributing.md

```diff
@@ -31,14 +31,17 @@ test suite which covers a variety of scenarios. Changes made to the code should
 not break existing features. We strongly recommend running the standard tests
 before you send a pull request to development. 
 
+If you add new features, you also need to add tests for functionality.
+Patches that do not contain tests for new features will not be accepted by the
```

Comment:
"If you add new features, you also need to add tests for functionality" ==> "If you add new features, you also need to add tests for these new features"

Reply:
The existing sentence is bad, agreed. But your suggestion is not much better.
I went with a pronoun in 58fca7e5a258488967f44fbfa051a76064c63cfb.